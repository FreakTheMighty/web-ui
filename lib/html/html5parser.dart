// Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
// for details. All rights reserved. Use of this source code is governed by a
// BSD-style license that can be found in the LICENSE file.
// Generated by scripts/tree_gen.py.

/**
 * An HTML5 parser written in Dart.
 *
 * Intended to parse all html documents using the [html5 parsing algorithm][p],
 * although the implementation is not yet complete.
 *
 * [p]: http://dev.w3.org/html5/spec/parsing.html#parsing
 */
#library('html5parser');

#source('htmltree.dart');
#source('parser.dart');
#source('tree.dart');
#source('token.dart');
#source('tokenkind.dart');
#source('tokenizer_base.dart');
#source('tokenizer.dart');
