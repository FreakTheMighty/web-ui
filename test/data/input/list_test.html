<!doctype html>
<!--
   Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
   for details. All rights reserved. Use of this source code is governed by a
   BSD-style license that can be found in the LICENSE file.
-->
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
  <title>list_test</title>
  <!-- use ../input/ so that the url also works from the output directory -->
  <script type='application/javascript' src="../input/testing.js"></script>
</head>
<body>
  <template iterate="item in loopItemList">
    <span>{{item}}</span>
  </template>
  <script type="application/dart">
    import 'dart:html';
    import '../input/common.dart';
    main() {
      window.setTimeout(() => window.postMessage('done', '*'), 0);
    }
  </script>
</body>
</html>
