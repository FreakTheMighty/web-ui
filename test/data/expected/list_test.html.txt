Content-Type: text/plain
layer at (0,0) size 800x600
  RenderView at (0,0) size 800x600
layer at (0,0) size 800x36
  RenderBlock {HTML} at (0,0) size 800x36
    RenderBody {BODY} at (8,8) size 784x20
      RenderInline {SPAN} at (0,0) size 7x19
        RenderText {#text} at (0,0) size 7x19
          text run at (0,0) width 7: "a"
      RenderText {#text} at (7,0) size 4x19
        text run at (7,0) width 4: " "
      RenderText {#text} at (0,0) size 0x0
      RenderInline {SPAN} at (0,0) size 8x19
        RenderText {#text} at (11,0) size 8x19
          text run at (11,0) width 8: "b"
      RenderText {#text} at (0,0) size 0x0
      RenderText {#text} at (0,0) size 0x0
      RenderText {#text} at (0,0) size 0x0
